### DISCUSSION TITLE
Unhandled false remark directive

### DISCUSSION THREAD

**Discussion by underdoeg** [Milkdown/milkdown#597] [General]
**Describe the bug**
Unhandled remark directive.
I am trying to add a custom remark directive plugin to some existing markdown pages. 

The syntax of the plugin will be ```:gallery-entry{id=20}``` and this works as expected.

The problem is, we have some pages with lists of mac addresses. ```D0:D9:4F:EB:F0:D3``` The :D3 part gets parsed as a remark directive but is ignored in my code:

```
 parseMarkdown: {
                match: (node) => {
                    return node.type === "textDirective" && node.name === "gallery-entry"
                },
                runner: (state, node, type) => {
                    state.addNode(type, {id: (node.attributes as { id: number }).id});
                },
            },
```
I suspect milkdown does not know what to do with this unhandled false directive and throws: ```Cannot match target parser for node:```

I did not see a way to change the behaviour of the parser (like use :: instead of : )

One possible solution would be to have milkdown fallback to a regular text node if the AST is unhandled and just display the content as is?

**To Reproduce**
Follow the example iFrame Plugin tutorial and try to parse a markdown document with the following contents:
```D0:D9:4F:EB:F0:D3```

**Expected behavior**

**Milkdown version**
6.2

**Desktop (please complete the following information):**

-   OS: Liunux
-   Browser Chromium
-   Version 102

**Comment by Saul-Mirone** [comment]
It's not a bug. You should take care of the extension you're using. Maybe this package can help:
https://www.npmjs.com/package/@ezone-devops/milkdown-plugin-directive-fallback

**Reply by underdoeg** [reply]
Ah yes. That is exactly what I was looking for.
